testing/internal/e2e/boundary/credential.go

```diff
@@ -202,10 +202,10 @@ func CreateStaticCredentialPrivateKeyCli(t testing.TB, ctx context.Context, cred
 	return credentialId, nil
 }
 
-// CreateStaticCredentialPasswordCli uses the cli to create a new password credential in the
+// CreateStaticCredentialUsernamePasswordCli uses the cli to create a new password credential in the
```

Comment:
@laero FYI: I'm making this change in this PR. Turns out we needed to also update another method name for something similar (UsernamePasswordDomain), and we're using these methods for some work I'm doing now, so I figured to just get this in.